### PR TITLE
fix(#7215)[Fault Management]: Fix shelving and acknowledging faults with single and bulk actions

### LINKED ISSUE #7215: [Fault Management] Cannot acknowledge or shelve faults with bulk actions
#### Summary
Users are currently unable to acknowledge or shelve faults in the fault management view with the bulk actions.

#### Expected vs Current Behavior
When selecting one or many faults in the fault management screen, the user should be able to shelve or acknowledge a fault. Today they do not 

#### Steps to Reproduce
1. Create an alarm or two
2. Navigate to the Fault Management View
3. Select the checkbox for all alarms
4. Note: the shelve and acknowledge buttons do not highlight

#### Environment
* Open MCT Version: 3.2.0
* Deployment Type: openmct-yamcs and viper

#### Impact Check List

- [ ] Data loss or misrepresented data?
- [x] Regression? Did this used to work or has it always been broken?
- [x] Is there a workaround available?
- [x] Does this impact a critical component?
- [ ] Is this just a visual bug with no functional impact?
- [x] Does this block the execution of e2e tests?
- [ ] Does this have an impact on Performance?

### PR DESCRIPTION
**Pull request by ozyx**
Closes #7215 

### Describe your changes:

- Makes `shelvedFaults` a computed property to fix Vue reactivity
- Fixes issue where acknowledging a single fault via the context menu while others were checked would acknowledge only the checked faults.
- Merges `FaultManagementListView` into `FaultManagementView` to simplify
- Adds some a11y
- Promote fault management e2e tests to stable and remove expect.softs

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [ ] Is this a [notable change](../docs/src/process/release.md) that will require a special callout in the release notes? For example, will this break compatibility with existing APIs or projects that consume these plugins?

### Author Checklist

* [x] Changes address original issue?
* [x] Tests included and/or updated with changes?
* [x] Has this been smoke tested?
* [x] Have you associated this PR with a `type:` label? Note: this is not necessarily the same as the original issue.
* [x] Have you associated a milestone with this PR? Note: leave blank if unsure.
* [ ] Is this a breaking change to be called out in the release notes?
* [ ] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [ ] Changes appear to address issue?
* [ ] Reviewer has tested changes by following the provided instructions?
* [ ] Changes appear not to be breaking changes?
* [ ] Appropriate automated tests included?
* [ ] Code style and in-line documentation are appropriate?

### CODE REVIEW

File: e2e/helper/faultUtils.js [hunk 1]
```diff
@@ -75,88 +77,95 @@ async function navigateToFaultItemInTree(page) {
...
  * @param {import('@playwright/test').Page} page
  */
-async function selectFaultItem(page, rowNumber) {
-  await page.locator(`.c-fault-mgmt-item > input >> nth=${rowNumber - 1}`).check();
+export async function selectFaultItem(page, rowNumber) {
+  await page
+    .getByLabel('Select fault')
+    .nth(rowNumber - 1)
+    .check({
+      // Need force here because checkbox state is changed by an event emitted by the checkbox
```
Comment (unlikelyzero) [inline e2e/helper/faultUtils.js:158]: gross

Comment (ozyx) [reply e2e/helper/faultUtils.js:158]: yep but i'd rather this than have to restructure all of fault management

File: e2e/tests/functional/plugins/faultManagement/faultManagement.e2e.spec.js [hunk 2]
```diff
@@ -75,180 +96,180 @@ test.describe('The Fault Management Plugin using example faults', () => {
...
-    expect.soft(await utils.getFaultName(page, 1)).toEqual(faultTwoName);
+    faultResultCount = await getFaultResultCount(page);
+    expect(faultResultCount).toEqual(1);
+    expect(await getFaultName(page, 1)).toEqual(faultTwoName);
 
     // all faults
-    await utils.clearSearch(page);
-    faultResultCount = await utils.getFaultResultCount(page);
-    expect.soft(faultResultCount).toEqual(5);
+    await clearSearch(page);
```
Comment (unlikelyzero) [inline e2e/tests/functional/plugins/faultManagement/faultManagement.e2e.spec.js:214]: what in tarnation

File: e2e/tests/functional/plugins/faultManagement/faultManagement.e2e.spec.js [hunk 3]
```diff
@@ -75,180 +96,180 @@ test.describe('The Fault Management Plugin using example faults', () => {
...
+    faultResultCount = await getFaultResultCount(page);
+    expect(faultResultCount).toEqual(5);
 
     // search triggerTime
-    await utils.enterSearchTerm(page, faultFiveTriggerTime);
+    await enterSearchTerm(page, faultFiveTriggerTime);
```
Comment (unlikelyzero) [inline e2e/tests/functional/plugins/faultManagement/faultManagement.e2e.spec.js:219]: ahhhhh

**Review by unlikelyzero — APPROVED** [review]
Approved and added https://github.com/nasa/openmct/issues/7561 to our backlog

**Pull request merged**